Tasks/DockerV2/dockerbuild.ts

```diff
@@ -71,5 +68,51 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     return dockerCommandUtils.build(connection, dockerFile, commandArguments, labelArguments, tagArguments, (data) => output += data).then(() => {
         let taskOutputPath = utils.writeTaskOutput("build", output);
         outputUpdate(taskOutputPath);
+
+        const builtImageId = getBuiltImageIdFromDockerBuiltOutput(output);
```

Comment:
Do we need image ID? Is tag not good enough for the use case?

Reply:
The same imageId can have multiple tags and could potentially collapse if we aggregate all of them in one place